client/admin/integrations/IntegrationsTable.js

```diff
@@ -78,18 +78,18 @@ export function IntegrationsTable({ type }) {
 		<Th key={'_createdBy'} direction={sort[1]} active={sort[0] === '_createdBy'} onClick={onHeaderClick} sort='_createdBy'>{t('Created_by')}</Th>,
 		isBig && <Th key={'_createdAt'} direction={sort[1]} active={sort[0] === '_createdAt'} onClick={onHeaderClick} sort='_createdAt'>{t('Created_at')}</Th>,
 		<Th key={'username'} direction={sort[1]} active={sort[0] === 'username'} onClick={onHeaderClick} sort='username'>{t('Post_as')}</Th>,
-	].filter(Boolean), [sort, isBig]);
+	].filter(Boolean), [sort, onHeaderClick, isBig, t]);
 
-	const renderRow = useCallback(({ name, _id, type, username, _createdAt, _createdBy: { username: createdBy }, channel = [] }) => {
-		const handler = useMemo(() => onClick(_id, type), []);
+	const renderRow = useCallback(function IntegrationRow({ name, _id, type, username, _createdAt, _createdBy: { username: createdBy }, channel = [] }) {
```

Comment:
Move `IntegrationRow` to the outer scope and use it like `const renderRow = useCallback((props) => <IntegrationRow {...props} />, [...])`.